src/main/config/CU_Checkstyle.xml

```diff
@@ -5,421 +5,205 @@
+        <module name="RedundantModifier"/>
+        <module name="RightCurly"/>
+        <module name="SimplifyBooleanExpression"/>
+        <module name="SingleSpaceSeparator">
+            <property name="validateComments" value="true"/>
+        </module>
+        <module name="StringLiteralEquality"/>
+        <module name="SuperClone"/>
+        <module name="SuperFinalize"/>
+        <module name="ThrowsCount"/>
+        <module name="TrailingComment"/>
+        <module name="TypecastParenPad"/>
+        <module name="UnnecessaryParentheses"/>
+        <module name="UnusedImports"/>
+        <module name="UpperEll"/>
+        <module name="WhitespaceAfter"/>
+        <module name="WhitespaceAround"/>
+        <!-- Cornell specific checks --> 
+        <module name="AnnotationUseStyle">
+      		<property name="severity" value="error"/>
+      		<property name="elementStyle" value="ignore"/>
+      		<property name="closingParens" value="ignore"/>
+      		<!-- "never" because Sun's compiler cannot handle a trailing comma and will not compile (who broke the build?) -->
+      		<property name="trailingArrayComma" value="never"/>
 		</module>
-		
-		<module name="IllegalType">
+	    <module name="IllegalType">
 			<property name="severity" value="warning"/>
 			<property name="illegalClassNames" value="java.util.GregorianCalendar, java.util.Hashtable, java.util.Vector"/>
 		</module>
-		
-		<module name="ImportControl">
-			<property name="severity" value="ignore"/>
-		</module>
-		
-		<module name="Indentation">
-			<property name="severity" value="ignore"/>
-			<property name="basicOffset" value="2"/>
-            <property name="braceAdjustment" value="0"/>
-            <property name="caseIndent" value="2"/>
-            <property name="throwsIndent" value="2"/>
-            <property name="lineWrappingIndentation" value="2"/>
-            <property name="arrayInitIndent" value="2"/>
-		</module>
-
-		<module name="InnerAssignment">
-			<property name="severity" value="info"/>
-		</module>
-		
-		<module name="InterfaceIsType">
-			<property name="severity" value="warning"/>
-		</module>
-		
-		<module name="LineLength">
-			<property name="severity" value="ignore"/>
-			<property name="max" value="160"/>
-            <property name="tabWidth" value="4"/>
-		</module>
-		
 		<module name="LocalFinalVariableName">
-			<property name="severity" value="warning"/>
 			<property name="format" value="^([a-z][a-zA-Z0-9_]*|[A-Z0-9_]*)$"/>
 		</module>
-		
-		<module name="LocalVariableName">
-			<property name="severity" value="warning"/>
-		</module>
-		
-		<module name="MagicNumber">
-			<property name="severity" value="ignore"/>
-		</module>
-		
-		<module name="MemberName">
-			<property name="severity" value="info"/>
-		</module>
-
-		<module name="MethodLength">
-			<property name="severity" value="info"/>
-			<property name="max" value="150"/>
-			<property name="countEmpty" value="false"/>
-		</module>
```

Comment:
Our Confluence documentation mentions that for this rule, we should "keep it commented." If doing so results in our configuration not being clean enough, then can a separate user story be created to potentially revisit this rule?

Reply:
I have created https://jira.cornell.edu/browse/KFSPTS-17981